cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DatastoreImpl.java

```diff
@@ -1857,10 +1850,10 @@ private String insertNewWinnerRevision(SQLDatabase db,DocumentBody newWinner,
         return newRevisionId;
     }
 
-    private void setCurrent(SQLDatabase db,DocumentRevision winner, boolean currentValue) {
+    private void setCurrent(SQLDatabase db, long sequence, boolean currentValue) {
```

Comment:
Regarding the other discussion around the naming of `currentValue` I can see how it could be confusing if taken in the context of something like a `checkAndSet`. The meaning here is clearly `valueOfCurrentToWrite` and that could be added in javadoc for the method or just renaming the field to `valueOfCurrent` to make it clear that current is the thing we're talking about and not some state of the value.